pkgs/by-name/te/termsonic/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildGoModule,
+  fetchzip,
+  pkg-config,
+  alsa-lib
+}:
+buildGoModule rec {
+  name = "termsonic";
+  version = "7a3aabee59e1a427aff755fc69759265ad9d0adc";
+  vendorHash = "sha256-wCtQD9f1mbN/0qUZnamPoVn9p4Ra5dQ34vlT+XjPF3k=";
+  src = fetchzip {
+    url = "https://git.sixfoisneuf.fr/termsonic/snapshot/termsonic-${version}.zip";
+    hash = "sha256-C5/4679qw4CAdUt9lXpPIR3yejrPdddvmjgbpLF3SvA=";
+  };
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ alsa-lib ];
+  strictDeps = true;
+  meta = with lib; {
+    homepage = "https://git.sixfoisneuf.fr/termsonic";
+    description = "A Subsonic client running in your terminal";
+    license = licenses.gpl3Plus;
+    platforms = with platforms; [
```

Comment:
This is a list of list! Use `platforms.unix` instead.

Reply:
I didn't even notice that :)